### PR TITLE
Adding in mmvec recipe

### PR DESCRIPTION
**Pull request by mortonjt**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Just as a heads up, this package has already been pushed to [conda forge](https://github.com/conda-forge/staged-recipes/pull/8464) - but we have changed the name (see [here](https://github.com/conda-forge/rhapsody-feedstock/issues/4)).  Not sure about the internals, but it looks like the easiest route is to create another recipe.

### CODE REVIEW

File: recipes/mmvec/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
-{% set version = "0.2.1" %}
-{% set name = "rhapsody" %}
+{% set version = "0.6.0" %}
+{% set name = "mmvec" %}
```
Comment (scopatz) [inline recipes/mmvec/meta.yaml]: Please don't modify other recipes

Comment (mortonjt) [reply recipes/mmvec/meta.yaml]: reverting

**Comment by mortonjt** [comment]
@conda-forge/help-python, I don't expect the windows builds to work due to skbio, but we should be good otherwise.

**Comment by mortonjt** [comment]
Hi @conda-forge/help-python, is there any chance that we could get some feedback on this?

File: recipes/mmvec/LICENSE [hunk 2]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (isuruf) [inline recipes/mmvec/LICENSE]: You don't need this file here if it's in the sdist

File: recipes/mmvec/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  entry_points:
+    - qiime2.plugins = mmvec.q2.plugin_setup:plugin
+
+requirements:
+  host:
+    - python >=3
+    - pip
+
+  run:
+
```
Comment (isuruf) [inline recipes/mmvec/meta.yaml]: Remove this empty line

File: recipes/mmvec/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+source:
+  url: https://pypi.io/packages/source/m/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b16d8e0bc7da9127e608759cbde02b074ebca6f8dffe250b0d7c94738171fcf5
+
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
```
Comment (isuruf) [inline recipes/mmvec/meta.yaml]: Same here

File: recipes/mmvec/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+requirements:
+  host:
+    - python >=3
+    - pip
+
+  run:
+
+    - python >=3
+    - ipython
```
Comment (isuruf) [inline recipes/mmvec/meta.yaml]: Why do you need `ipython` here?

Comment (mortonjt) [reply recipes/mmvec/meta.yaml]: gone

File: recipes/mmvec/meta.yaml (hunk 3 above)
Comment (mortonjt) [reply recipes/mmvec/meta.yaml]: gone

File: recipes/mmvec/meta.yaml (hunk 4 above)
Comment (mortonjt) [reply recipes/mmvec/meta.yaml]: removed

File: recipes/mmvec/LICENSE (hunk 2 above)
Comment (mortonjt) [reply recipes/mmvec/LICENSE]: removed

Comment (isuruf) [reply recipes/mmvec/LICENSE]: The file is still there

Comment (mortonjt) [reply recipes/mmvec/LICENSE]: misread. The file is now gone.

**Comment by mortonjt** [comment]
@isuruf , @scopatz, thank you for your reviews.  I just wanted to confirm if the LICENSE is still a requirement for the CI build.

We're hoping to have a major release soon, so your feedback is greatly appreciated!

**Comment by mortonjt** [comment]
Thanks!

**Pull request merged**